addons/mass_mailing/models/mailing_trace.py

```diff
@@ -62,6 +62,7 @@ class MailingTrace(models.Model):
              'However the ID is needed for several action and controllers.',
         index='btree_not_null',
     )
+    mail_message_id = fields.Many2one(related='mail_mail_id.mail_message_id')
```

Comment:
I think this one must be compute (otherwise the compute method is used only if you install mass_mailing_sms)